mlos_bench/mlos_bench/config/schemas/environments/common-environment-subschemas.json

```diff
@@ -15,7 +15,7 @@
                         "type": "string"
                     },
                     "uniqueItems": true,
-                    "minItems": 1
+                    "minItems": 0
```

Comment:
resolves an issue where we test for explicitly listing an empty tunable_params group `[]`